docs/deployment/README.md

```diff
@@ -245,6 +245,31 @@ You should see output like:
 Terraform has been successfully initialized!
 ```
 
+#### Import existing resources
```

Comment:
Obviously this all breaks if you subsequently delete the resources you imported. However, since we introduced deletion protection for data resources a few releases ago for the Kes deployment, that shouldn't happen by default even if people delete their old Cloudformation stacks. I tested that myself. 

Beyond that, it seemed kind of obvious that you shouldn't delete these resources once you have imported them, but I'm happy to add a note if others think it's worthwhile

Reply:
Added b9eeba60f46c4b6c758ffde6477267761d286544 just to be safe